kbase-extension/static/kbase/js/common/cellComponents/filePathWidget.js

```diff
@@ -260,27 +285,23 @@ define([
             let layout = renderLayout();
             container.innerHTML = layout.content;
             layout.events.attachEvents(container);
-
-            places = {
-                parameterFields: ui.getElement(`${cssClassType}-fields-row`)
-            };
         }
 
         // EVENTS
-        function attachEvents() {
```

Comment:
This wasn't being called anywhere, and I am not sure what it was used for. I think it may be important, but am unsure so I commented it out.

Reply:
Not sure either, the original paramsWidget had a call to this function in `start`, after the initial `renderParameters() call` it resolves with a `then(function() { attachEvents(): })....` 

still not entirely clear on what we need as a base level for the message bus on each widget, and what can be handled by the parent vs not. curious to see what @briehl has to say